src/main/resources/templates/categories/ventilation-units/ventilationUnitsSubCategory.ftl

```diff
@@ -0,0 +1,11 @@
+<#include '../../layout.ftl'>
+
+<@defaultPage>
+  <@form.govukForm "/categories/ventilation-units/">
```

Comment:
I also think this was missing on the Lamps example, but this should use the Reverse Router to generate the path, rather than having it hard coded.